Nurse Tasks/Preview Content/jsonAPI.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  jsonAPI.swift
+//  Nurse Tasks
+//
+//  Created by Wu Maggie on 2023-11-16.
+//
+
+import Foundation
+public struct Task:Codable{
+    var clientName: String
+    var assignedTo : String
+    var location: Address
+    var startTime, endTime: Date
```

Comment:
Can you please add 'reminderTime' as well

Reply:
For the reminder, after I tried, I think it doesn't need the 'reminderTime', cause if the task will be finished after one month, how can we set the reminderTime? How about we set it automatically reminder one day or 30 mins before the due day, or give some option same as apple event reminder

Reply:
I just think we need the variable to be a datetime value, because if we want to give a reminder 5 minutes before it's due we will need that information. We can just have a default of 5 or 10 minutes before the due date or something, the user won't choose the reminder period yet

Reply:
Maybe you're right about using 'apple reminders' see my comments in the group chat. It looks complicated though :)

Reply:
How about we can show an alert when the task due, which 'endTime' == current time, and add more feature later if when have time. Then we can add the 'reminderTime' in the future